packages/office-ui-fabric-react/src/components/MessageBar/examples/MessageBar.Basic.Example.tsx

```diff
@@ -2,295 +2,122 @@ import * as React from 'react';
-    <MessageBar
-      styles={{
-        root: {
-          background: 'rgba(0, 178, 148, 0.2)'
-        },
-        dismissSingleLine: {
-          border: '1px solid #5c2d91'
-        },
-        expandSingleLine: {
-          border: '1px solid #e3008c'
-        },
-        dismissal: {
-          selectors: {
-            '& .ms-Button-icon': {
-              color: '#004b50'
-            }
-          }
-        },
-        expand: {
-          selectors: {
-            '& .ms-Button-icon': {
-              color: '#004b50'
-            }
-          }
-        },
-        innerText: {
-          textDecoration: 'underline'
-        }
-      }}
-      messageBarType={MessageBarType.blocked}
-      isMultiline={false}
-      onDismiss={log('test')}
-      dismissButtonAriaLabel="Close"
-      truncated={true}
-      overflowButtonAriaLabel="See more"
-    >
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi luctus, purus a lobortis tristique, odio augue pharetra metus, ac
-      placerat nunc mi nec dui. Vestibulum aliquam et nunc semper scelerisque. Curabitur vitae orci nec quam condimentum porttitor et sed
-      lacus. Vivamus ac efficitur leo. Cras faucibus mauris libero, ac placerat erat euismod et. Donec pulvinar commodo odio sit amet
-      faucibus. In hac habitasse platea dictumst. Duis eu ante commodo, condimentum nibh pellentesque, laoreet enim. Fusce massa lorem,
-      ultrices eu mi a, fermentum suscipit magna. Integer porta purus pulvinar, hendrerit felis eget, condimentum mauris.{' '}
-      <Link href="www.bing.com">Visit our website.</Link>
-    </MessageBar>
-
-    <Label>Severe warning, multiline, with action buttons - custom styles</Label>
-    <MessageBar
-      styles={{
-        root: {
-          background: 'rgba(227, 0, 140, 0.2)'
-        },
-        icon: {
-          color: '#e3008c'
-        },
-        actions: {
-          background: 'rgba(180, 0, 158, 0.4)'
+    <StackItem>
+      <Label>SevereWarning MessageBar - defaults to multiline, with action buttons</Label>
+      <MessageBar
+        messageBarType={MessageBarType.severeWarning}
+        actions={
+          <div>
+            <MessageBarButton>Yes</MessageBarButton>
+            <MessageBarButton>No</MessageBarButton>
+          </div>
         }
-      }}
-      messageBarType={MessageBarType.severeWarning}
-      actions={
-        <div>
-          <MessageBarButton styles={{ root: { color: '#e3008c' } }}>Yes</MessageBarButton>
-          <MessageBarButton styles={{ root: { color: '#e3008c' } }}>No</MessageBarButton>
-        </div>
-      }
-    >
-      <span>Lorem ipsum dolor sit amet, a elit sem interdum consectetur adipiscing elit.</span>
-      <Link href="www.bing.com">Visit our website.</Link>
-    </MessageBar>
+      >
+        <span>Severe warning lorem ipsum dolor sit amet, a elit sem interdum consectetur adipiscing elit.</span>
```

Comment:
We might want to use `Text` instead of `span` here.

This also applies to the other uses of `span` throughout this change.

Reply:
In these cases I don't think we actually need either a span or Text, it can just be inline text. (previously I'd just changed the indentation, not really looked at the example contents)